client/my-sites/customer-home/cards/features/domain-upsell/index.jsx

```diff
@@ -67,7 +72,8 @@ export default function DomainUpsell( { context } ) {
 			isP2Site( primarySite ) ||
 			isNotAtomicJetpack( primarySite ) );
 
-	const shouldNotShowMyHomeUpsell = ! isProfileUpsell && ( siteDomainsLength || ! isEmailVerified );
+	const shouldNotShowMyHomeUpsell =
+		! isProfileUpsell && ( siteDomainsLength || ! isEmailVerified || isSiteSetupComplete );
```

Comment:
Wondering if not showing for unverified email users is a valid reason, since users could take this opportunity to validate their email and maybe upsell the domain. Not related to this specific change, but it could help us drive more result. WDYT?

Reply:
I think is good, do you want to comment about it here (pb5gDS-2Q9-p2#comment-3680)?